.gitignore

```diff
@@ -1,5 +1,6 @@
 bin
 */*.iml
+.DS_Store
```

Comment:
🤔

Reply:
It's a hidden file generated by mac os which bothers me for a long time. 😕I can delete this as well.

Reply:
> which bothers me for a long time

😢